pages/dkp/kaptain/2.1.0/tutorials/mlflow/index.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: layout.pug
+navigationTitle: MLflow
+title: MLflow
+menuWeight: 18
+excerpt: Use MLflow with Kaptain
+beta: false
+enterprise: false
+---
+
+You can use the MLflow platform with Kaptain to collect data and manage the lifecycle of your Machine Learning and Artificial Intelligence models and experiments. MLflow supports distributed architectures, where the tracking servers, backend store, and artifact store can reside in remote hosts. When using MLflow with Kaptain, Percona handles the backend store and MinIO handles the artifact storage per default. The tracking server is handled by Kaptain.
+
+With MLflow, you are able to:
+
+- Gather and track data related to experiments for analysis.
+- Package ML/AI code and share it with other data scientists or transfer to production environments.
+- Manage and deploy models from ML libraries to model serving and inference platforms.
+- Manage the full lifecycle of an MLflow model from a centralized store (model versioning, stage transitions, annotations, etc.)
+
+## Prerequisites
+
+- You have launched a Jupyter notebook with Kaptain. (Using another notebook would require you to build a custom image for this purpose)
+
+  <p class="message--important"><strong>IMPORTANT: </strong>If you decide to use your own custom image or an image not provided by Kaptain to run a distributed training job, create an image following the (<a href="../sdk/image-builder/">recommended container image creation steps</a>. Also ensure MLflow has been installed on it.</p>
+
+- Optional: If you are using SDK, you have [set up Kaptain SDK for your desired training framework](https://docs.d2iq.com/dkp/kaptain/2.0.0/tutorials/sdk/).
+
+## Set up MLflow
+
+Starting from this release, MLflow is available in every notebook per default. You do not need to download, nor install it.
```

Comment:
```suggestion
Starting with Kaptain 2.1.0, MLflow is available in every notebook per default. You do not need to download, nor install it.
```

Reply:
We usually avoid using version numbers whenever we can, so we don't need to touch the pages between releases. However, you made me realize we should not use wordings like "starting from this release" either 🙈 
```suggestion
MLflow is available in every notebook per default. You do not need to download, nor install it.
```